webapp/utils/hemi.ts

```diff
@@ -68,13 +70,30 @@ export const getHemiStatusOfBtcDeposit = ({
           vaultStateAddress,
         }),
       ),
-  ]).then(([hemiAwareOfBtcTx, confirmed]) =>
-    hemiAwareOfBtcTx
-      ? confirmed
-        ? BtcDepositStatus.BTC_DEPOSITED
-        : BtcDepositStatus.READY_TO_MANUAL_CONFIRM
-      : BtcDepositStatus.BTC_TX_CONFIRMED,
-  )
+    // check the number of confirmations of the deposit
+    getBitcoinKitAddress(hemiClient).then(bitcoinKitAddress =>
+      hemiClient
+        .getTxConfirmations({
+          bitcoinKitAddress,
+          txId: deposit.transactionHash,
+        })
+        // api throws if the tx Id is not found, which is the same as 0 confirmations from our POV.
+        .catch(() => 0),
+    ),
+  ]).then(function ([hemiAwareOfBtcTx, confirmed, confirmations]) {
+    if (!hemiAwareOfBtcTx) {
+      return BtcDepositStatus.BTC_TX_CONFIRMED
+    }
+    if (confirmed) {
+      return BtcDepositStatus.BTC_DEPOSITED
+    }
+    // Hemi requires 6 confirmations before a deposit is manually processed. If that doesn't happen
+    // it should allow the user to manually confirm the deposit.
+    if (confirmations > BitcoinConfirmationsOnHemi) {
```

Comment:
@max-sanchez  can you confirm that if confirmations are `> 6`, we can allow users to manually confirm the bitcoin deposit? Or should it be `>= 6` ?

Reply:
After getting no response from @max-sanchez , I went with Gabriel suggestion of considering the 6th confirmation too (So using `>=`). Applied here [bac8897](https://github.com/hemilabs/ui-monorepo/pull/873/commits/bac88978e8ad2a3e1f6f0579d925d657fb8cadb5)